chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -293,6 +293,9 @@ pane.item.related.count.singular = %S related:
 pane.item.related.count.plural  = %S related:
 pane.item.parentItem			= Parent Item:
 
+drag.invalid.title 				= Invalid Item
+drag.publications.onlyTopLevel	= Only top-level Zotero items are allowed in My Publications
```

Comment:
I thought they were called Zotero items. I don't think there really is a good user-friendly name for them unless you do some sort of brandish thing with them, like Zoterios™, but that's just silly.